ios/Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 63cb0d0aef536c8cbf15bf664fcfe31852a13dd1
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.7.0
```

Comment:
Is this upgrade necessary?  If not, please revert.

Reply:
This has been reverted, thanks!